mozdownload/scraper.py

```diff
@@ -819,6 +819,9 @@ def binary_regex(self):
 
     def build_filename(self, binary):
         """Return the proposed filename with extension for the binary."""
+        if hasattr(self, 'builds'):
```

Comment:
Maybe we should better initialize the `builds` property to `[]` in the constructor so that it always exist.